flink-runtime/src/main/java/org/apache/flink/runtime/filecache/FileCache.java

```diff
@@ -150,9 +150,10 @@ public void shutdown() {
 			for (File dir : storageDirectories) {
 				try {
 					FileUtils.deleteDirectory(dir);
+					LOG.info("removed file cache directory {}", dir.getAbsolutePath());
 				}
 				catch (IOException e) {
-					LOG.error("File cache could not properly clean up storage directory.");
+					LOG.error("File cache could not properly clean up storage directory: {}", dir, e);
```

Comment:
I would suggest to make the directory information symmetric: Either `dir.getAbsolutePath()` or `dir` in both logging  statements.